### PR TITLE
kebab case some arguments in LocusWalker and Spark walkers

### LINKED ISSUE #5478: Kebab case spark walker arguments
A number of arguments in the SparkWalker classes were not kebab cased.  They should be made into kebab case.

### LINKED ISSUE #5479: LocusWalker --maxDepthPerSample should be kebabed
The argument `maxDepthPerSample` in locus walker should be made kebab case.  The same problem exists in `LocusWalkerSpark`

### PR DESCRIPTION
**Pull request by davidbenjamin**
Closes #5479.  Closes #5478.

@lbergelson here you go.

### CODE REVIEW

**Comment by samuelklee** [comment]
Thanks @davidbenjamin!  Couple more in AssemblyRegionReadShardArgumentCollection?

**Comment by davidbenjamin** [comment]
@samuelklee They're in this PR (that's the #5478 part).

File: src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
  */
 public abstract class LocusWalker extends GATKTool {
 
-    @Argument(fullName = "maxDepthPerSample", shortName = "maxDepthPerSample", doc = "Maximum number of reads to retain per sample per locus. Reads above this threshold will be downsampled. Set to 0 to disable.", optional = true)
+    @Argument(fullName = "max-depth-per-sample", shortName = "max-depth-per-sample", doc = "Maximum number of reads to retain per sample per locus. Reads above this threshold will be downsampled. Set to 0 to disable.", optional = true)
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java]: There are two error messages that refer to the old `maxDepthPerSample` one in LocusWalker.getDownsamplingInfo() and the other in LocusWalkerSpark.  Could you extract a constant from this and have all three locations use it?

**Comment by samuelklee** [comment]
Oops, duh, so they are---didn't realize it from the title alone and misread the diff between this and another one of my branches!

**Comment by samuelklee** [comment]
Also @lbergelson please disregard that second review request!  Man am I looking foolish on this branch.

File: src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java (hunk 1 above)
Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java]: done

**Pull request merged**